integration/setup/error.go

```diff
@@ -9,6 +9,38 @@ import (
 	"github.com/giantswarm/microerror"
 )
 
+var executionFailedError = &microerror.Error{
+	Kind: "executionFailedError",
+}
+
+// IsExecutionFailed asserts executionFailedError.
+func IsExecutionFailed(err error) bool {
+	return microerror.Cause(err) == executionFailedError
+}
+
+var notExistsError = &microerror.Error{
+	Kind: "notExistsError",
+}
+
+// IsNotExists asserts notExistsError.
+func IsNotExists(err error) bool {
+	c := microerror.Cause(err)
+
+	if c == nil {
+		return false
+	}
+
+	if strings.Contains(c.Error(), "does not exist") {
```

Comment:
Why this is needed?

Reply:
I copied this from the tccp resource. This is apparently necessary when searching for instances, like we do with the master instance and the bastion instance. Does that answer your question?

Reply:
Together with other comment that answers my question.